### DISCUSSION TITLE
Can't switch after update to tasmota 13.4.0.4

### DISCUSSION THREAD

**Discussion by MyUniversalDeveloper** [arendst/Tasmota#21165] [Q&A]
### PROBLEM DESCRIPTION
After update my devices (ESP8285N08, ESP8266EX) to tasmota 13.4.0.4 it is impossible to switch any power outlet. By clicking on the buttons of the web interfaces I can hear a short relay switch but the state turns immediately to off. This happens also by using the physical buttons.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): _____
- [x] Tasmota binary firmware version number used: _____
  - [x] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment]
You did not fill in **any** of the details needed to document the status and operation of Tasmota.

**Comment by MyUniversalDeveloper** [comment]
I can't edit this site anymore. Therefore I create a new discussion.

**Comment by sfromis** [comment]
And don't open an "issue" just because you have trouble using the thing. Better ask about such in "Discussions" (or on Discord)
